### PR TITLE
Upload an 'EthicalResearch' image and format the figure using MyST

### LINKED ISSUE #1640: [BUG] Maintaining Consistency in the Guide for Ethical Research
# Summary of Issue 

_The Turing Way_ project is an open-source endeavour that empowers contributors from around the world to leverage their skills, knowledge and expertise to build and expand its content.
_The Turing Way_ guides are evolving with multiple chapters co-developed by individuals from varied backgrounds, who are all passionate about sharing knowledge around data science and research.
To sustain and support this vibrant community, _The Turing Way_ book must remain consistent and accessible as it evolves. 
Although the [style guide](https://the-turing-way.netlify.app/community-handbook/style.html) chapter already outlines guidelines for keeping a consistent style across the book, not all chapters follow these suggestions as they have been written asynchronously by different authors.

We have curated a consistency checklist that chapters in _The Turing Way_ need to pass to help us maintain a consistent style across the book. However, we need your help to push this forward!

For reference, the checklist is discussed in the chapter on [maintaining consistency](https://the-turing-way.netlify.app/community-handbook/consistency.html) in _The Turing Way_. There you can also find useful instructions on how to address these issues

## What needs to be done

This issue focuses on maintaining consistency in the [Guide for Ethical Research](https://the-turing-way.netlify.app/ethical-research/ethics-intro.html). To help with this, we invite contributors to do the following:

- **Guide Location**: [book/website/ethical-research](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/website/ethical-research)
- **Latest commit when this issue was created**: 8c0a340

#### File Specific Checklist

[Landing Page](https://the-turing-way.netlify.app/ethical-research/ethical-research.html)
- [ ] Format image using MyST - see the style guide for [recommendations](https://the-turing-way.netlify.app/community-handbook/style/style-figures.html)
- [x] Add this label `(er)=` to the top of the file
- [ ] Check for proper grammar and spelling

[Ethical Decisions in Preclinical Research](https://the-turing-way.netlify.app/ethical-research/preclinical-research.html)
- [ ] Chapter is incomplete. Raise an issue for content to be written for the chapter and for it to be modularised. See the [chapter template](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/templates/chapter-template) for guidance

#### How you can help
1. Please pick a consistency check from the checklist above (check it off from the issue description)
2. Comment below what consistency check you would like to work on
3. Work on your contributions locally or on Github
4. Raise a PR for your contributions to be reviewed and integrated in _The Turing Way_ repo!

### Who can help

Anyone who would like to make the _The Turing Way_ as consistent and polished as possible. We especially welcome new contributors to try their hand on this!

For any questions, comments, suggestions, or feedback, please comment below :muscle:

### PR DESCRIPTION
**Pull request by ufangYang**
### Summary

Fixes #1640

*Format image using MyST
* upload the figure

### List of changes proposed in this PR (pull-request)

* * Format the figure using MyST and add the short description 
* * Upload the EthicalResearch.jpg

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/ethical-research/ethics-intro.md [hunk 1]
```diff
@@ -12,6 +12,15 @@ Below is a table summarising these questions.
 | **Who researchers interact with** | With research subjects (for example: do we treat them with respect?) | With fellow researchers (for example: do we adhere to a code of conduct?) |
 | **What shapes research** | Shaped by society (for example: how is AI as a solution generally understood?) | Shaped by institutional norms (for example: what research questions are being funded?) |
 
+```{figure} ../figures/EthicalResearch.jpg
+---
+height: 500px
+name: Ethical Research
```
Comment (malvikasharan) [inline book/website/ethical-research/ethics-intro.md:18]:
```suggestion
name: EthicalResearch 
```

**Review by malvikasharan — APPROVED** [review]

**Review by paulowoicho — COMMENTED** [review]
Hi @ufangYang! Thanks for the PR! The `MyST` formatting looks good, but the image is in the wrong folder, so the image does not show up in the Netlify preview of this PR. 

The image needs to be in [this folder](https://github.com/ufangYang/the-turing-way/tree/master/book/website/figures) of your fork of the repository. I've made a commit to reflect this!

**Comment by malvikasharan** [comment]
Looks great @ufangYang. I have just removed a space between the two words for the image name. Thank you so much. Let me know if you are happy for us to merge this one.

**Comment by malvikasharan** [comment]
@all-contributors please add @ufangYang for bug fix.

**Review by paulowoicho — APPROVED** [review]

**Comment by malvikasharan** [comment]
Thank you @paulowoicho - I had totally missed the location.

**Comment by ufangYang** [comment]
@malvikasharan @paulowoicho thank you all!

**Pull request merged**